server/scsynth/SC_Bela.cpp

```diff
@@ -198,23 +160,19 @@ void SC_BelaDriver::BelaAudioCallback(BelaContext* belaContext) {
 
         float* inBuses = mWorld->mAudioBus + mWorld->mNumOutputs * bufFrames;
         float* outBuses = mWorld->mAudioBus;
-        int32* inTouched = mWorld->mAudioBusTouched + mWorld->mNumOutputs;
-        int32* outTouched = mWorld->mAudioBusTouched;
 
         int minInputs = sc_min(numInputs, mWorld->mNumInputs);
         int minOutputs = sc_min(numOutputs, mWorld->mNumOutputs);
 
         int anaInputs = 0;
-        if (numInputs < (int)mWorld->mNumInputs) {
-            anaInputs = sc_min(belaContext->analogInChannels, (int)(mWorld->mNumInputs - numInputs));
+        if (numInputs < mWorld->mNumInputs) {
+            anaInputs = sc_min(belaContext->analogInChannels, static_cast<int>(mWorld->mNumInputs - numInputs));
         }
         int anaOutputs = 0;
-        if (numOutputs < (int)mWorld->mNumOutputs) {
-            anaOutputs = sc_min(belaContext->analogOutChannels, (int)(mWorld->mNumOutputs - numOutputs));
+        if (numOutputs < mWorld->mNumOutputs) {
+            anaOutputs = sc_min(belaContext->analogOutChannels, static_cast<int>(mWorld->mNumOutputs - numOutputs));
         }
 
-        int bufFramePos = 0;
-
         // THIS IS TO DO LATER -- LOOK AT CACHEING AND CONSTING TO IMPROVE EFFICIENCY
```

Comment:
i also wanted to ask, should this comment be here? i'm not sure what it's saying, and it references jack functions that i don't think would belong if they were uncommented...